src/screens/SiteDetailScreen.js

```diff
@@ -114,6 +114,74 @@ class SiteDetailScreen extends Component {
         });
     };
 
+    renderAlternateSites = () => {
+        const {selectedSite} = this.props;
+        const {sectionTitleStyle, textStyle, bottomMargin} = styles;
+
+        if (selectedSite && selectedSite.alternateSites) {
+            return (
+                <View>
+                    <Text style={sectionTitleStyle}>
+                        {campsite_form.alternate_sites}
+                    </Text>
+                    <Text style={[textStyle, bottomMargin]}>
+                        {selectedSite.alternateSites}
+                    </Text>
+                </View>
+            );
+        }
+
+    };
+
+    renderCellProvider = ({cellProvider}) => {
+        if (cellProvider) {
+            const {textStyle, bottomMargin} = styles;
+
+            return (
+                <Text style={[textStyle, bottomMargin]}>
+                    {campsite_form.cell_provider_options[cellProvider]}
+                </Text>
+            );
+        }
+    };
+
+    renderCellStrength = ({cellProvider, cellStrength}) => {
+        if (cellStrength) {
+            const {textStyle, bottomMargin, leftPad} = styles;
+            const stylesList = cellProvider ? [textStyle, bottomMargin, leftPad] : [textStyle, bottomMargin];
```

Comment:
If there's also a `cellProvider`, then put some padding next to the `cellStrength`. If not, then just leave it as-is with its regular styling.